src/Lean/Elab/Tactic/Repeat.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2022 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Scott Morrison
+-/
+
+import Lean.Elab.Tactic.Basic
+import Lean.Meta.Tactic.Cases
+
+open Lean Elab Tactic
+
+/--
+`iterate n tac` runs `tac` exactly `n` times.
+`iterate tac` runs `tac` repeatedly until failure.
```

Comment:
Perhaps this should be removed, as `repeat` already does this? Or alternatively, can this be folded into `repeat` so we have one tactic rather than two?

Reply:
I've made an internal TODO.